rune/libenclave/internal/runtime/pal/pal_linux.go

```diff
@@ -6,6 +6,7 @@ import (
 	"github.com/go-restruct/restruct"
 	"github.com/inclavare-containers/epm/pkg/epm-api/v1alpha1"
 	"github.com/inclavare-containers/rune/libenclave/attestation"
+	_ "github.com/inclavare-containers/rune/libenclave/attestation/sgx/challenger"
 	_ "github.com/inclavare-containers/rune/libenclave/attestation/sgx/ias"
```

Comment:
This line is still necessary?

Reply:
> This line is still necessary?

Yes, because `sgx-epid-challenger.go` has a init function, other packages(sgx challengers) must import it to do it's init function.

Reply:
I mean the **ias** package is still required to be imported in this way?

Reply:
> I mean the **ias** package is still required to be imported in this way?

ok, **ias** package should be imported by rune/libenclave/attestation/sgx/sgx-challenger/sgx-epid-challenger.go, so it's not imported here.